examples/04_manipulating_images/plot_resample_to_template.py

```diff
@@ -10,34 +10,32 @@
 
 ###############################################################################
 # First we load the required datasets using the nilearn datasets module.
-from nilearn.datasets import fetch_localizer_button_task
+from nilearn.datasets import fetch_neurovault_ids
```

Comment:
Why are we important fetch_neurovault_ids and using fetch_neurovault_motor_task below?